libs/postgres_ffi/wal_craft/src/lib.rs

```diff
@@ -37,35 +37,35 @@ pub static REQUIRED_POSTGRES_CONFIG: Lazy<Vec<&'static str>> = Lazy::new(|| {
 });
 
 impl Conf {
-    pub fn pg_distrib_dir(&self) -> PathBuf {
+    pub fn pg_distrib_dir(&self) -> Result<PathBuf> {
```

Comment:
NIT: let's not shadow `Result` from the implicitly imported stdlib prelude and use `anyhow::Result` instead, for readability.
Don't change it now if you don't want to, but please consider it in the future.